app/assets/javascripts/proxy_status.js

```diff
@@ -1,9 +1,10 @@
 //= require proxy_status/puppet
 //= require proxy_status/logs
+//= require common/tabs_helper
```

Comment:
moved to an helper, can you test with `proxy_status` too?

Reply:
how about moving it to tfm.tools while you're at (and to es6) it instead so it can also be used by other pages directly, and leave the triggers inside the current actions?

Reply:
Yep, sounds good